PLATEXP-3524: Add new blog banner

ssalinas24 opened the pull request:
https://rackspace.atlassian.net/browse/PLATEXP-3524

This PR does the following:
- Upload new CFCF Rings Background Image svg
- Upload new Expert Insight tagline svg
- Update CSS to use the new images

Testing Instructions:

- Use the deploy-preview for this PR, but add the route/url to the blog. So ultimately visit this URL..
https://deploy-preview-347--docs-landing-page.netlify.app/blog
- Use the browser developer tools or inspector to view the HTML
- Find the stylesheet link element and change href to remove the main domain
<img width="1017" alt="Screen Shot 2023-02-28 at 11 24 07 PM" src="https://user-images.githubusercontent.com/14296817/222057409-6aa4bfde-0d75-4a09-bcfa-d72c7c4373d5.png">

- Just leave it set to /style.css so it is a relative link and using the stylesheet from the build of this PR instead of the production docs.rackspace.com URL.

## After:
<img width="1680" alt="Screen Shot 2023-03-01 at 12 01 44 AM" src="https://user-images.githubusercontent.com/14296817/222057878-d5b2f4eb-ecee-4301-85dd-dc3a17745c44.png">

ssalinas24 commented:
@lfielder the only problem I have left to resolve is the left alignment of the logo. It is currently going all the way to the left because it is part of the background image. Ideally we need it to left align to the body content as the designs show. Which is tricky when using as multiple background images of one container. I'm still trying to figure out a workaround or hack. Like trying to get the logo to be the background image of the `<br>` element via CSS from this PR and repo.

Usually it's easier if the logo was inside the content area of the container or another child div/element inside the container. However, the inner content of the blog-header-container is not part of this repo. It is part of the technical-blog at that point. 
https://github.com/rackerlabs/technical-blog/blob/master/layouts/partials/components/headline.html
Which means we would need two different PRs and the live site would be incomplete without both PRs at the same time.

ssalinas24 commented:
## Then initial Design they provided
[Rackspace-Technical-Blog-Header-BRA-TSK-8123.pdf](https://github.com/rackerlabs/docs-landing-page/files/10866497/Rackspace-Technical-Blog-Header-BRA-TSK-8123.pdf)

## Current Preview:
@jkinley @lfielder follow the test instructions from the initial comment and you will see the latest preview.
It will look like this. The remaining issue is trying to achieve the left alignment still. @jkinley let me know if i'm missing an easy fix here. @lfielder if not then perhaps we can get them to compromise and allow me to center the item. That should be way easier.

![Screen Shot 2023-03-01 at 6 27 10 PM](https://user-images.githubusercontent.com/14296817/222298541-04997136-6000-4701-a2fa-2bde57dd52c4.png)

jkinley commented:
@ssalinas24 I am sure I can help with this, but its difficult from just reviewing the screenshots. Maybe we can sync and look at the code? I don't know a lot about the history here, but here are a few thoughts:

1. The background image is decoration and is fine as a background image. 
2. The 'insights' logo is NOT decoration, and shouldn't be an image. That important 'heading level text' is 'trapped' in an image. I'd have the text and icon as elements and real content.
3. If you are stuck using that as a background image, as a quick fix background positioning should work.

ssalinas24 commented:
@jkinley Thanks for taking a look. I'm new to these repos as well, and unsure of how many other repos use the same classes, because this repo is a layout template for several other docs repos. The technical-blog is just one of them. So figuring i'm stuck keeping this as a background image. I tried updating the logo to an H1 and the CSS has additional code for H1 situations already.

So I inspected all the HTML elements that had left padding or margin. They were the following.
15px
4rem (aka. 56px)
15px
15px on .container
79.578px from auto margin on .container while in a 1680px browser width
.blog-post has 2rem margin (aka. 28px)

So I totaled that up and converted to a percentage of my browser/screen width of 1680px
Which equates to 0.107486. So I just rounded down to 10.7% and added in the last two commits for the blog-headline-container padding. This would allow the padding to be dynamic and match any viewport. This hopefully helps avoid updating all media query paddings.

I guess I could also leave the padding as it was and add a `background-position: 10.7%;` instead and would be the same thing.

The deploy preview has been updated. If you follow the test instructions in the initial comment then you can see it live in your browser.

jkinley commented:
> 15px 4rem (aka. 56px) 15px 15px on .container 79.578px from auto margin on .container while in a 1680px browser width .blog-post has 2rem margin (aka. 28px)
> 
> So I totaled that up and converted to a percentage of my browser/screen width of 1680px Which equates to 0.107486. So I just rounded down to 10.7%

We should really avoid magic numbers like this whenever we can. This might work now, but its potentially going to be a bear for someone to figure out in the future. If you must do this for now, we should put these values in comments and make a note to come back and fix it later. Again, I am happy to look at the code with you.

ssalinas24 commented on assets/pages/page-blog.scss:
```diff
@@ -186,6 +150,18 @@ img[src$="cloudformation-launch-stack.png"] {
 
+.blog-headline-container::after {
+  content: '.';
+  background-image: url(https://deploy-preview-347--docs-landing-page.netlify.app/img/expert-insights-logo-tagline.svg);
```
put relative path back

jkinley reviewed (approved):
LGTM. Given the circumstances (as I suspected), it best to do as minimal as possible here to make things work. Go any deeper and you are just going to be falling down a rabbit hole.
